api/ElementInternals.json

```diff
@@ -0,0 +1,2213 @@
+{
+  "api": {
+    "ElementInternals": {
+      "__compat": {
+        "support": {
+          "chrome": {
+            "version_added": "77"
+          },
+          "chrome_android": {
+            "version_added": "77"
+          },
+          "edge": {
+            "version_added": "79"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "64"
+          },
+          "opera_android": {
+            "version_added": "55"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": "12.0"
+          },
+          "webview_android": {
+            "version_added": "77"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "ariaAtomic": {
```

Comment:
These aren't from the spec linked in the description, they're from https://w3c.github.io/aria/#ARIAMixin.

Can you split them out into a separate PR and add them to Element.json at the same time?

Chrome 81 as the version is confirmed by https://storage.googleapis.com/chromium-find-releases-static/dbe.html#dbef8d4838807211e2940f0929c8109cdf06b9d5

Reply:
Ack, thought I excluded these.  Removing now!